front/pages/api/w/[wId]/assistant/agent_configurations/[aId]/history/index.ts

```diff
@@ -7,7 +7,10 @@ import { isLeft } from "fp-ts/lib/Either";
 import * as reporter from "io-ts-reporters";
```

Comment:
Meta the name of the endpoint is not ideal, is that history or feedbacks?

Reply:
Ah actually I see this is the history of versions not the feedback. 

I guess the name is fine 👍